packages/aragon-wrapper/src/cache/index.js

```diff
@@ -3,6 +3,8 @@ import memoryStorageDriver from 'localforage-memoryStorageDriver'
 import { Subject } from 'rxjs/Rx'
 import { concat } from 'rxjs/observable/concat'
 
+const trackedKeys = new Set()
```

Comment:
This will be shared across cache instances, which could leak change values when two instances are cleared. 

Any reason for not making this a class property?

Reply:
Ohh good point; mostly so I could hide it but let's make it a class property 👍

Reply:
Made it in a private class field in 940efbc9ea08b476f6e0ade49b79af38dca90b03 (just learned today that `babel@7.1.0` implemented this 🎉 )